PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,40 @@
 ## High-level description
 
+<!-- This template uses HTML comments for providing guidance. They are not visible in the PR output -->
+
+
 What features does this change enable? What bugs does this change fix?
 
 
-## Corresponding DC/OS tickets (obligatory)
+## Corresponding DC/OS tickets (required)
+
+<!--
+
+Please provide a list of JIRA tickets that this Pull Request acts upon. This section will be parsed, 
+a comment will be added, and the FixVersion field of the JIRA will be set to DC/OS release version upon merged.
+The FixVersion will correspond to the [fix_version_map](https://github.com/dcos/dcos/blob/master/mergebot-config.json#L30) 
+of the mergebot-config.
+
+These DC/OS JIRA ticket(s) must be updated, in the moment this PR lands. 
 
-These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:
+-->
 
-  - [DCOS_OSS-<number>](https://jira.mesosphere.com/browse/DCOS_OSS-<number>) Foo the Bar so it stops Bazzing.
+  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.
 
 
 ## Related tickets (optional)
 
-Other tickets related to this change:
+<!--
 
-  - [DCOS_OSS-<number>](https://jira.mesosphere.com/browse/DCOS_OSS-<number>) Foo the Bar so it stops Bazzing.
+Other tickets related to this change. The JIRAs mentioned in this section will not be parsed, no comments be will added,
+FixVersion will not be updated.
 
+Please keep the header '## Related tickets (Optional)' if you are adding optional tickets. Information below this header 
+will excluded from Mergebot parsing behavior for JIRA comment updates and fix version.
 
-## Checklist for all PRs
+-->
 
-  - [ ] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
-  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
```

Comment:
Did we remove this on purpose?

CC @adamtheturtle

Reply:
😢

Reply:
Yes, this checkbox was constantly being ignored upon during PR creation and reviews. We should probably bring this up during reviews, address it with our review policy protocol rather than having a checkbox for every PR. We could list this policy requirement in the wiki if needed.

That said, please share some positive aspects of having this checkbox. If there is a thinking that this checkbox is still useful, let us include it again.

Reply:
I added this back to the template.